.github/actions/build_ci/entrypoint.sh

```diff
@@ -74,8 +74,17 @@ build_zephyr(){
     	cd ./zephyr &&
     	source zephyr-env.sh &&
 	cd ../.. &&
-	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr &&
-	cd build-zephyr &&
+	echo  "###### Build for qemu_cortex_m3 ######" &&
+	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr-m3 &&
+	cd build-zephyr-m3 &&
+	make VERBOSE=1 &&
+	echo  "###### Build for qemu_riscv64 ######" &&
```

Comment:
probably unrelated to the failure but we need to get out from `build-zephyr-m3` before starting the building for `riscv` targets.

Reply:
uhm, probably not unrelated because it's working for me locally.

Reply:
Thanks for pointing out the issue!
What i can figure out is why it did not failed when i tested on my github (https://github.com/arnopo/libmetal/runs/4700942696?check_suite_focus=true)
i will fix it

Reply:
Still something to fix.